### ISSUE TITLE
Show Remote Content function has a bug.

### ISSUE THREAD

**Issue by odelphi** [thunderbird-conversations/thunderbird-conversations#1743]
Thunderbird version: 91.2.1
Thunderbird Conversation version: 4.0.5
Operating System: Win10

When I have this addon enabled,  I open an email, I get the "Remote Content was Hidden" banner, which is fine (that is what I want). It gives me the option to "Show Remote Content".  I click on that.  When I click on it, the content briefly (for about half a second) shows the remote content then the email goes completely blank and the address bar shows:
chrome://conversations/content/stub.html?urls=imap-message%3A%2F%2F<redacted content>%40<redacted content>%2FINBOX%234501
By the way, when I open ANY email I get that URL in the address bar. Is that correct?   

Randomly, the "show remote content" option still displays (sometimes it does, sometimes it doesn't).  When I click on that, it finally shows my email with the remote content.  The address bar still displays the above URL.  I redacted the part of the URL that shows my email address.  If I close and reopen the email, the remote content displays.

When the "show remote content" bar doesn't continue to display I have no choice but to close the email and open it again.  Randomly, when I reopen the email, it is still blank AND there is no "Show Remote Content" bar.  Through trial and error, I learned there is a little icon on the right that has a tool tip that says "Show content in a new tab".  When I click that, it randomly shows my email and the remote content.  Sometimes, though, the email content is still blank. Instead, it displays the message:  "We're sorry, but we can't display this conversation, because the messages have been moved or deleted."  I close the email and reopen it.  It is still blank, but when I click the "Show message in new tab" icon, it show the email the the remote content.

Is that how it supposed to work or is this a random bug?  I have a feeling this is not how it supposed to work.  When I disable this addon, everything works perfect, so I know it is this addon.

**Comment by Standard8** [comment]
Unfortunately I can't reproduce this, I've tried a few things including remote content with links to broken images and they all seem to work fine.

I guess it could be timing related. Though maybe also check if other add-ons could be causing an issue or not.

It might be worth checking the error console to see if there's any errors turn up when viewing these messages. You can find it under the three-bar menu -> Tools -> Developer Tools.

**Comment by odelphi** [comment]
OK.  I checked my other addons.  It appears one of my other addons is interfering with this addon.  I have the addon "Mail Hops" which is an addon that tells me where the mail originated - helps determining junk mail, phishing attempts, etc.
Are they incompatible with each other because this addon converts my email to a browser HTML with the URL: chrome://conversations/content/stub.html?urls=imap-message%3A%2F%2F%40%2FINBOX%234501, which causes Mail Hops to blow a gasket?  Mail Hops probably doesn't know how to handle that.  I like Mail Hops for the security aspect, so I will probably have to disable Conversations until they are compatible.

**Comment by Standard8** [comment]
I've just tried installing Mail Hops, and I don't see any issues with it - apart from the fact its message header buttons don't work (related to #1665).

Getting the output as I previously requested might be useful.

**Comment by odelphi** [comment]
Here is the error log after I enabled the Conversations Addon and the Mailhop addon. (All other addons were disabled).  I then opened an email that had remote content (pictures) and hit the "Show Remote Content" button.  The content showed for about 2 seconds, then the whole email went blank.

![Firefox Error Console](https://user-images.githubusercontent.com/52181093/140654829-70f7150c-8cbb-46af-bfdb-4f42e3975436.png)
